README.md

```diff
@@ -481,8 +483,28 @@ negotiator.authenticate({
 	unsignedToken,
 });
 
-negotiator.on("proof", () => {
+negotiator.on("proof", ({ proof, issuer, error }) => {
+	// the proof will be received here (valid or failed)
+});
+```
+
+### Authenticate ownership of multiple off chain Tokens
+
+Authenticating ownership of the tokens will provide a list of proof data with a limited expiry.
+
+```javascript
+/**
+ * @param {{issuer: String, unsignedToken: Object}[]} tokens object array of tokens
+ */
+negotiator.authenticate([{
+	issuer,
+	unsignedToken,
+}]);
+
+negotiator.on("proof", ({ issuers, issuersValidated }) => {
```

Comment:
Hi, @nicktaras , I think better use different event name, like "multiproof", because 2 events with same name and different return data can confuse people and can force mistakes in code

Reply:
I would like to in version 3.0.0 provide the same interface for Single and Multi. Where 2.8.0 and 2.9.0 will use proof and for version 3.0.0 there will only be multi - regardless of having 1 or more tokens we can use the same interface and delete much of our code.

Reply:
Having flows for single and multi are too complicated I feel - there should be as fewer processes and learnings for the end developer and users.

Reply:
Updated.

Reply:
maybe you forgot to push your updates, because in manual it still use 
negotiator.on("proof",... in both scenarios, but input data are different.

Reply:
Sorry it looks like I'd not responded to the correctly. This was intentional, so for version 3.0.0 we can move to one implementation for proof - to return an array.